src/chart/Scatter.js

```diff
@@ -0,0 +1,171 @@
+    Scatter.prototype.publish("dotRepresentation", "cross", "set", "Selects the representation for the data", ["circle", "rectangle", "cross"]);
+    Scatter.prototype.publish("pointSize", 8, "number", "pointSize");
+
+    Scatter.prototype.updateChart = function (domNode, element, margin, width, height) {
+        var context = this;
+
+        this._palette = this._palette.switch(this.paletteID());
+
+        var column = this.svgData.selectAll(".data")
+            .data(this._data, function (d) { var k = d.filter(function(d, i) {return i > 0;}); return k + Math.random(); })
+        ;
+
+        column.enter().append("g")
+            .attr("class", "data")
+        ;
+
+        column
+            .each(function (d, i) {
+                var element = d3.select(this);
+                var point = element.selectAll(".point").data(d.filter(function(d, i) {return i > 0;}));
+                var point1 = element.selectAll(".point").data(d.filter(function(d, i) {return i > 0;}));
+                switch (context.dotRepresentation()) {
+                    case "circle" :
+                        var title = point
+                          .enter().append("circle")
+                            .attr("class", "point")
+                            .attr("r", context.pointSize()/2)
+                            .on("click", function (d2, idx) {
+                                context.click(context.rowToObj(d), context._columns[idx + 1]);
+                            })
+                            .append("title")
+                        ;
+                        break;
+
+                    case "rectangle" :
+                        var title = point
+                          .enter().append("rect")
+                            .attr("class", "point")
+                            .attr("width", context.pointSize())
+                            .attr("height", context.pointSize())
+                            .on("click", function (d2) {
+                                context.click(context.rowToObj(d), context._columns[idx + 1]);
+                            })
+                            .append("title")
+                        ;
+                        break;
+
+                    case "cross" :
+                        var title = point
+                          .enter().append("line")
+                            .attr("class", "point")
+                            .attr("stroke-width", 2)
+                            .on("click", function (d2) {
+                                context.click(context.rowToObj(d), context._columns[idx + 1]);
+                            })
+                            .append("title")
+                        ;
+                        var title1 = point1
+                            .enter().append("line")
+                            .attr("class", "point")
+                            .attr("stroke-width", 2)
```

Comment:
If I move this to style sheet, I will have to color the stroke with every other shape, and the size estimation we have now, will also change. Also this is only relevant for the cross shape.